plugins/plugin-pullrequest-parent/che-plugin-pullrequest-ide/src/main/resources/org/eclipse/che/plugin/pullrequest/client/ContributeMessages.properties

```diff
@@ -142,4 +142,13 @@ step.authorize.codenvy.on.vcs.host.error.cannot.access.vcs.host.content=Your bro
 #
 contributor.extension.error.updating.contribution.attributes=An error occurred while updating contribution attributes to the current project: {0}.
 contributor.extension.error.setOriginRepository=An error occurred while setting up origin repository: {0}.
-contributor.extension.default.commit.description={0}: {1}
\ No newline at end of file
+contributor.extension.default.commit.description={0}: {1}
+
+#
+# Contribution panel stab content
+#
+stub.text.project.not.provide.vsc=Project does not provide any VSC.
```

Comment:
`does not have` seems will be better, project actually doesn't provide anything, it may has something.